swagger/swagger.json

```diff
@@ -389,6 +392,15 @@
           "description" : "Amount in pence. Can't be more than the available amount for refunds",
           "minimum" : 1.0,
           "maximum" : 1.0E7
+        },
+        "refund_amount_available" : {
+          "type" : "integer",
+          "format" : "int32",
+          "example" : "200000",
+          "description" : "Amount in pence. It should be the available amount for refund",
```

Comment:
also here make explicit whether it's the amount before or after the refund has been performend